README.md

```diff
@@ -0,0 +1,54 @@
+[![CurseForge](https://cf.way2muchnoise.eu/full_667299_downloads.svg)](https://curseforge.com/minecraft/mc-mods/yacl)
+
+[![Ko-fi](https://ko-fi.com/img/githubbutton_sm.svg)](https://ko-fi.com/isxander)
+
+Yet Another Config Lib, like, what were you expecting?
+
+</div>
+
+## Why does this mod even exist?
+
+This mod was made to fill a hole in this area of Fabric modding. The existing main config libraries don't achieve what I want from them:
+
+- **[Cloth Config API](https://modrinth.com/mod/cloth-config)**:<br/>It's stale. The developer of cloth has clarified that they are likely not going to add any more features. They don't want to touch it.
+- **[SpruceUI](https://github.com/LambdAurora/SpruceUI)**:<br/>It isn't designed for configuration. In this essence the design feels cluttered. Further details available in this issue.
+- **[MidnightLib](https://modrinth.com/mod/midnightlib)**:<br/>It's mostly bloat and doesn't have a focus on config. This library includes other unwanted features such as cosmetics.
+- **[OwoLib](https://modrinth.com/mod/owo-lib)**:<br/>It's content focused. Like MidnightLib, it does a lot of other things as well, adding to the size.
+
+As you can see, there's sadly a drawback with all of them and this is where YetAnotherConfigLib comes in.
+
+## How is YACL better?
+
+YACL has the favour of hindsight. Whilst developing this fresh library, I can make sure that it does everything right:
+
+- It's just a config library. YACL contains no other features, just config screen generation.
+- It's lightweight. YACL leaves managing your config up to you, it doesn't contain an alternative to [AutoConfig](https://shedaniel.gitbook.io/cloth-config/auto-config/introduction-to-auto-config-1u) or similar. (Can be paired with my other library, [Settxi](https://github.com/isXander/Settxi), for this feature)
+- Easy API. YACL takes inspiration from [Sodium's](https://modrinth.com/mod/sodium) internal configuration library.
+- It's styled to fit in Minecraft. YACL's GUI is designed to fit right in.
```

Comment:
Could you make the first sentence of all these points in bold? I think it highlights the main point so people don't need to read the full explanation.

Reply:
Done @isXander

Reply:
I mean the how yacl is better section